### ISSUE TITLE
CCVideoPlayer: ability to attach under GLView

### ISSUE THREAD

**Issue by psineur** [cocos2d/cocos2d-iphone-extensions#5]
Subj, there was a discussion on cocos2d forum. Here it is:  
http://www.cocos2d-iphone.org/forum/topic/9220

**Comment by psineur** [comment]
Addition: implemented for gles20 branch: http://www.cocos2d-iphone.org/forum/topic/29478

**Comment by richardgroves** [comment]
I think the code in http://www.cocos2d-iphone.org/forum/topic/29478 will work on GL-ES1.1 too - it is the older path where

if ([_theMovie respondsToSelector: @selector(view)]) returns NO 

that will definitely not work yet. As far as I can tell I don't use anything that is GL2.0 only - that is just the cocos that I work with now, so I can't say it will work on 1.1.

**Comment by psineur** [comment]
Yeah, it's ok with GLES 1.1, but there can be API diffs in cocos2d-iphone 2.x & 1.x
Like this:

```
    [[CCDirector sharedDirector].view
```

In 1.x there's no view property in CCDirector, only openGLView

It's pretty easy to support both  with some ifdefs

**Comment by richardgroves** [comment]
doh! Guess there will be these subtle issues between things as cocos v2 seems to be more than just a move to OpenGL ES2.0.
